routes/index.coffee

```diff
@@ -2,5 +2,6 @@ express = require 'express'
 router = express.Router()
 
 router.get /^\/users\/(\d+)\/(atom|rss)$/, require './users'
+router.get /^.*\/favicon.ico$/, require './favicon'
```

Comment:
It *should* be located in the root path. However, [as this article reports](https://perishablepress.com/block-favicon-url-404-requests/), there seem to be some entities that look for it deeper down. So I thought it could not hurt to satisfy them, too, as this will reduce the number of failed requests.